src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/AddAzureRmVmssDataDiskCommand.cs

```diff
@@ -69,6 +69,11 @@ public partial class AddAzureRmVmssDataDiskCommand : Microsoft.Azure.Commands.Re
             ValueFromPipelineByPropertyName = true)]
         public string CreateOption { get; set; }
 
+        [Parameter(
+            Mandatory =false,
+            ValueFromPipelineByPropertyName = true)]
+        public string DeleteOption { get; set; }
```

Comment:
use argument completer to give accepted values. look for examples

Reply:
Done, thanks